docs/_static/custom.css

```
@@ -1,395 +1,738 @@
-/* Orion Finance SDK Documentation Theme */
-/* Ported from Orion Docs Docusaurus Config */
+/* Orion Finance SDK docs, brand.orionfinance.ai
+   Sea Blue #0D3AA9 · Navy #051743 · Off White #FBF8EF · Off Black #141418
+   EB Garamond (headers) · Satoshi (body) · IBM Plex Mono (code)
+*/
 
-/* --- FONTS --- */
 @import url('https://api.fontshare.com/v2/css?f[]=satoshi@300,400,500,600,700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://api.fontshare.com/v2/css?f[]=satoshi@300,400,500,600,700&display=swap')" to be "'https://api.fontshare.com/v2/css?f[]=satoshi@300,400,500,600,700&display=swap'" (import-notation)

(import-notation)